### PR TITLE
Update deploy-applications.md

### PR DESCRIPTION
**Pull request by PanuSaukko**
### Summarize the change in the pull request title

Describe your change, specifically *why* you think it's needed.

Fixes #Issue_Number (if necessary)

### CODE REVIEW

File: sccm/apps/deploy-use/deploy-applications.md [hunk 1]
```diff
@@ -115,7 +115,10 @@ One of the following approval settings appears, depending upon your version of C
...
+- **An administrator must approve a request for this application on the device**: Starting in version 1802, the administrator approves any user requests for the application before the user can install it on the requested device. If the administrator approves the request, the user is only able to install the application on that device. The user must submit another request to install the application on another device. This option is grayed out when the deployment purpose is **Required**, or when you deploy the application to a device collection.
+
+Starting in version 1810, you can also define a list of email addresses of users, who will be notificed about the approval requests.
```
Comment (aczechowski) [inline sccm/apps/deploy-use/deploy-applications.md]: Just a couple of edits here for style:

```suggestion
 Starting in version 1810, you can also define a list of email addresses to notify about the approval request. 
```

**Comment by PRMerger7** [comment]
Microsoft authors of articles need to make updates through the private repository so the updates can be staged and validated by the current validation rules. After you review staging and fix all validation issues, you can add the sign-off comment to let the PR reviewers know the updates are ready to be merged.It's easy to access the private repo version of your articles in GitHub. Just add -pr to the URL, as follows: 
 Public repo URL: https://github.com/Microsoft/azure-docs/blob/master/articles/azure-portal-overview.md 
 Private repo URL: https://github.com/Microsoft/azure-docs-pr/blob/master/articles/azure-portal-overview.md 
 We have closed this pull request. Please resubmit the updates through the private repo. If you need perms to the private repo, [see the contributor guide instructions](https://github.com/Microsoft/azure-docs/blob/master/contributor-guide/tools-and-setup.md#permissions)

**Comment by aczechowski** [comment]
Ugh, automation fail. Reopening

**Comment by aczechowski** [comment]
Thanks @PanuSaukko, this looks good.
#sign-off 
*Once merged*, this should be live with the next daily publishing process. Right now that *should* be tomorrow (Tuesday, 19 December) around 11:00 Pacific time. No further action needed from you.

**Pull request merged**